### DISCUSSION TITLE
git log does not show any output from prepare-commit-msg script

### DISCUSSION THREAD

**Discussion by JuliusHenke** [evilmartians/lefthook#355] [Q&A] [answered]
I would like to report an issue I stumbled upon, which I think is a bug:

When using a bash script for a `prepare-commit-msg` hook, no `echo` output of the script is shown in the lefthook log. I created a repo to reproduce this: https://github.com/JuliusHenke/lefthook-prepare-commit-msg-issue

Maybe I am missing something, but since it works with a script for `pre-commit`, I think this is not intended behaviour.

**Comment by mrexox** [comment, marked as answer]
Hey! Yes, this is a bug. Initially it was a hack because lefthook uses `prepare-commit-msg` hook for auto synchronization, but we don't want to see the output of lefthook on every commit, so we suppressed the output. But if you manually specify `prepare-commit-msg` hook, there is no need to suppress the output I think. I'll put it on my radar. 

To workaround this for now you can use set the `LEFTHOOK_VERBOSE=true` when running `git commit` or `lefthook run ...`
